src/openrct2/localisation/Localisation.h

```diff
@@ -112,6 +112,11 @@ class Formatter
         return CurrentBuf;
     }
 
+    auto GetStartBuf() const
```

Comment:
Unrelated?

Reply:
No. Take a look at `DrawTextBasic` and `DrawTextEllipsised`. They need to get the start buffer. If I just call `Buf()`, it will crash the game.